README.md

```diff
@@ -824,15 +824,17 @@ variable in your project, when dbt searches for implementations of a dispatched
 `dbt_utils` macro, it will search through your listed packages _before_ using
 the implementations defined in `dbt_utils`.
 
-Set the variable:
+Set the property `var` in `dbt_project.yml`:
```

Comment:
You're right to specify that this should be in `dbt_project.yml`. I'd rather refer to this as a "variable" vs. a "property," since [the latter word](https://docs.getdbt.com/reference/declaring-properties) has a different contextual meaning.
```suggestion
Set a variable in your `dbt_project.yml`:
```